README.md

```diff
@@ -12,6 +12,7 @@ The following resources are included:
 
 * [mysql/basic](./humanitec-resource-defs/mysql/basic): A basic MySQL.
 * [postgres/basic](./humanitec-resource-defs/postgres/basic): A basic Postgres.
+* [redis/basic](./humanitec-resource-defs/postgres/basic): A basic Redis.
 
 The `humanitec-resource-defs` directory includes the respective resource definitions.
```

Comment:
Unrelated, but could you also add an example in the `./examples` folder?

Reply:
Will create another PR for the same